integrations/access/datadog/client.go

```diff
@@ -58,6 +58,9 @@ type Datadog struct {
 	// simpler to integrate with the existing framework. Consider using the official
 	// datadog api client package: https://github.com/DataDog/datadog-api-client-go.
 	client *resty.Client
+
+	// TODO: Remove clientUnstable once on-call API is merged into official API
```

Comment:
IS there a link or something describing the API endpoint? Somewhere we can check if the API has been moved out unstable. I failed to find public documentation about this endpoint.

Reply:
Not that I'm aware of. This API endpoint has only been shared with us via email communication. I think we'll just need to link to the general API references page and keep an eye out for when documentation becomes available.